src/frontend/src/btc/schedulers/btc-wallet.scheduler.ts

```diff
@@ -30,6 +37,19 @@ export class BtcWalletScheduler implements Scheduler<PostMessageDataRequestBtc>
 		});
 	}
 
+	private async loadBtcAddress({
+		identity,
+		bitcoinNetwork
+	}: {
+		identity: OptionIdentity;
+		bitcoinNetwork: BitcoinNetwork;
+	}): Promise<string> {
+		// Save address for next timer
+		this.btcAddress = await getBtcAddress({ identity, network: bitcoinNetwork });
```

Comment:
Why is `this.btcAddress` needed?

Reply:
`btcAddress` is needed to fetch BTC address data (`loadBtcAddress`). And `this.btcAddress` stores the value so it can be reused in the next timer runs.

Reply:
I see, thanks for the explanation. Should/can we implement a similar pattern as the other workers by using a store?

For example in `ic-wallet.scheduler`.

```
private store: IcWalletStore<T> = {
		balance: undefined,
		transactions: {}
	};
```

Should we also need to store the transactions?